var/spack/repos/builtin/packages/findutils/package.py

```diff
@@ -45,7 +45,8 @@ def url_for_version(self, version):
     version('4.1.20', sha256='8c5dd50a5ca54367fa186f6294b81ec7a365e36d670d9feac62227cb513e63ab')
     version('4.1',    sha256='487ecc0a6c8c90634a11158f360977e5ce0a9a6701502da6cb96a5a7ec143fac')
 
-    patch('nvhpc.patch', when='%nvhpc')
+    patch('nvhpc.patch', when='@4.6.0 %nvhpc')
+    patch('nvhpc-long-width.patch', when='@4.8.0:4.8.99 %nvhpc')
```

Comment:
Can you add a brief comment on top of these patches, to explain what they do?

Reply:
descriptions added